NeuroID/src/test/java/com/neuroid/tracker/NIDJobServiceManagerTest.kt

```diff
@@ -31,7 +31,7 @@ class NIDJobServiceManagerTest {
         injectMockedApplication()
         assertEquals(NIDJobServiceManager.isSetup, true)
         assertEquals(NIDJobServiceManager.clientKey, "clientKey")
-        assertEquals(NIDJobServiceManager.endpoint, "endpoint")
+        assertEquals(NIDJobServiceManager.endpoint, "https://test.com")
```

Comment:
shouldn't this fail because it should use the constant?

Reply:
Had to fix testGetApiService() as well since this is the one that was setting it. Running testStartJob() by itself had no issue but since testGetApiService() was setting this to http://test.com, this was making it pass. Once testGetApiService() was patched with proper endpoint, testStartJob() started to fail. This was also patched so we are good now.